sunxspex/sunxspex_fitting/io.py

```diff
@@ -0,0 +1,137 @@
+"""
```

Comment:
Absolutely. I was just being cautious when I made this other io file since it was specific to instrument files and I was trying to avoid changing the code outside of the fitting stuff.